gemrb/core/TestRunner.cpp

```diff
@@ -0,0 +1,2 @@
+#define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
+#include "doctest.h"
```

Comment:
I guess this is just a remnant from the other test.

Reply:
Yeah, ignore this outside of the doctest branch.